static/app/views/app/index.tsx

```diff
@@ -1,5 +1,4 @@
 import {lazy, Profiler, Suspense, useCallback, useEffect, useRef} from 'react';
-import {useHotkeys} from 'react-hotkeys-hook';
```

Comment:
does this mean we no longer need this library?

Reply:
@davidenwang Yup, should I just drop it and push a commit here

Reply:
nah it's okay that can be a separate commit since I see there is another usage of the `react-hotkeys-hook`. I'm also trying to understand the difference between your new hook and the existing one we're using.

I guess if we were to use the pre-existing one, we would just have to call the `useHotkeys` hook in the search component one time for each unique search bar hotkey right? Is that the main reason we're writing our own now?

Reply:
The main benefit of this one is being able to define the available functions and their hotkeys in one place like here https://github.com/getsentry/sentry/blob/e01d19062cf224b8bb409242b210256aeff52fc8/static/app/components/smartSearchBar/types.tsx#L56-L73 which I think is cleaner than either placing each hotkey combination in its own hook or having to put the hooks in a loop